apps/frontend/src/pages/MyListings.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Typography, useMediaQuery, useTheme } from "@mui/material";
+import AccountSidebar from "../components/AccountSidebar";
+
+const MyListings = () => {
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("sm"));
+
+  return (
+    <>
+      {isDesktop && <AccountSidebar selectedItem="My Listings" />}
```

Comment:
Soooo, if this isn't shown on mobile, how does the user navigate then?

Reply:
There will be a hamburger in the navbar to those pages.

Navbar is still wip. Should be present on all pages except Login, Create Account, Forget Password, etc pages